web/index.html

```diff
@@ -70,287 +69,350 @@ <h3 class="sub-header">
+                <div class="toast" v-if="showImportToast" v-on:click.self="showImportToast=false;">
+                    <div class="toast-body">
+                        <span class="close-button topright" @click="showImportToast=false">&times;</span>
+                        <h2 style="margin-top: 0px; text-align: center;">Import your Infinite Craft data</h2>
+                        <p>
+                            <b>Step 1:</b>
+                            Click and drag this bookmarklet to your bookmarks bar
+                        </p>
+                        <div class="draggable-wrapper" :class="{ 'show-outline': isDraggingBookmarklet }"
+                            style="margin-left: auto; margin-right: auto;">
+                            <a :href="`javascript:(function(){${bookmarkletData}})();`" draggable="true"
+                                @dragstart="dragStart" @dragend="dragEnd" @click.prevent="preventClick"
+                                class="draggable" :class="{ 'dragging': isDraggingBookmarklet, 'shake': isShaking}">
+                                Infinite Craft Wiki
+                            </a>
+                        </div>
+                        <p>
+                            <b>Step 2:</b>
+                            Open <a class="text-link" href="https://neal.fun/infinite-craft/" target="_blank">Infinite
+                                Craft</a> by clicking on the bookmarklet once
+                        </p>
+                        <p>
+                            <b>Step 3:</b>
+                            Import your elements to The Wiki by clicking on the bookmarklet again
+                        </p>
+                        <div v-if="saveData.size != 0" class="button" @click="clearSaveData()">
+                            Clear User Data
+                        </div>
+                    </div>
+                </div>
+            </transition>
+
             <transition name="slide-up" mode="out-in">
                 <div key=0 v-if="currentElement">
                     <button class="controls left-arrow" :style="`top:max(${scroll}px, 300px); outline: none`"
                         @click="slideLeft"></button>
                     <button class="controls right-arrow" :style="`top:max(${scroll}px, 300px); outline: none`"
                         @click="slideRight"></button>
                     <transition name="slide-right" mode="out-in">
-                        <h2 :key="currentElement" style="text-align: center;" @click="navigateTo('')">
-                            <div class="item" style=" padding: 13px 15px 12px">
+                        <div :key="currentElement" style="text-align: center;" @click="navigateTo('')">
+                            <div class="item" style="transform: scale(1.5); margin-top:30px; margin-bottom: 30px; border-radius: 3px; border-width: 0.4px" :class="{'item-used': !saveData.has(currentElement)}">
```

Comment:
Switched out the way that the top element is rendered. It used to be rendered as a header which messed with the proportions and spacing a bit but looked alright, but adding the red triangle in the corner when it wasn't crafted yet and that no longer worked well, so it is now being rendered as a properly scaled version of the original, but with some minor tweaks to the border to make it look a little cleaner.